python_modules/libraries/dagster-shared/dagster_shared/serdes/serdes.py

```diff
@@ -416,24 +415,29 @@ def __whitelist_for_serdes(klass: T_Type) -> T_Type:
                 field_serializers=field_serializers,
             )
             return klass  # type: ignore
-        elif issubclass(klass, BaseModel) and (
-            serializer is None or issubclass(serializer, PydanticModelSerializer)
-        ):
-            whitelist_map.register_object(
-                klass.__name__,
-                klass,
-                serializer or PydanticModelSerializer,
-                storage_name=storage_name,
-                old_storage_names=old_storage_names,
-                storage_field_names=storage_field_names,
-                old_fields=old_fields,
-                skip_when_empty_fields=skip_when_empty_fields,
-                skip_when_none_fields=skip_when_none_fields,
-                field_serializers=field_serializers,
-            )
-            return klass  # type: ignore
         else:
-            raise SerdesUsageError(f"Can not whitelist class {klass} for serializer {serializer}")
+            from pydantic import BaseModel
```

Comment:
def want a comment for this one since usually we just defer to top of fn but this is intentionally later